src/components/Cards.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Card from "react-bootstrap/Card";
+import './Cards.css';
+
+export default function Cards() {
```

Comment:
nem helyes filename, Cards azt sejteti hogy egy listát jelenít meg a komponens, ez nem, ez egyetlen kártya komponens
```suggestion
export default function Card() {
```